README.md

```diff
@@ -256,6 +256,22 @@ To get in touch with the developers,
 [join us on Slack](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g)
 or [create an issue](https://github.com/trixi-framework/Trixi.jl/issues/new).
 
+## Institutional Support
+Trixi.jl relies on institutions that provide infrastructure for coding and testing, as well as encourage students, post-docs and scientists to contribute code. Institutes and groups that supported us in the past and present are listed below.
```

Comment:
We try to keep the line length limited to make it easier to review changes in git. You suggested using a "meaningful order". I'm not sure which order would be best. We could start with a chronological order since that requires the least amount of changes when updating the list in the future. However, I am of course open to other suggestions.

```suggestion
Trixi.jl relies on institutions that provide infrastructure for coding and testing,
as well as encourage students, post-docs, and scientists to contribute code.
Some institutes and groups that supported us in the past and present are listed below,
roughly in chronological order.
```